pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -649,6 +651,11 @@ func (cm *containerManagerImpl) Start(node *v1.Node,
 	return nil
 }
 
+func (cm *containerManagerImpl) reconcileState() {
+	cm.cpuManager.ReconcileState()
```

Comment:
We should check feature gate here it is possible that cpu manager is desabled.
BTW, my idea was migrating whole reconcile logic to container linux manager ,
https://github.com/kubernetes/kubernetes/blob/ef446db5e20e363034145afde5776ad5b1663a6f/pkg/kubelet/cm/cpumanager/cpu_manager.go#L306
https://github.com/kubernetes/kubernetes/blob/ef446db5e20e363034145afde5776ad5b1663a6f/pkg/kubelet/cm/cpumanager/cpu_manager.go#L349

and calling out `removeStaleState()` in topology manager
As is: https://github.com/kubernetes/kubernetes/blob/ef446db5e20e363034145afde5776ad5b1663a6f/pkg/kubelet/cm/cpumanager/cpu_manager.go#L295
To be: https://github.com/kubernetes/kubernetes/blob/ef446db5e20e363034145afde5776ad5b1663a6f/pkg/kubelet/cm/topologymanager/topology_manager.go#L173

May I do this next week?
Perhaps I can get it done on Monday or Tuesday.
(It seems I'm gonna open PR against your branch to add commit to this PR.)

Reply:
I see what do you mean, I just do not sure that it will improve readability and also will improve the performance of the kubelet(I can see container_manager_linux) calls additional go routines.